src/Http/StringStream.php

```diff
@@ -66,7 +66,7 @@ public function __toString()
     /**
      * Implements the PSR7 StreamInterface and closes the underlying stream.
      *
-     * @return void
+     * @return bool
```

Comment:
The definition in PSR7 declares that this needs to return void.

https://www.php-fig.org/psr/psr-7/#34-psrhttpmessagestreaminterface

Reply:
Phan marks it as incorrect because it actually returns a bool. Should we change what the function returns to match PSR or suppress the phan warning?

Reply:
we should change what it returns to match the standard. (It should be pretty easy, check if it's false and throw an exception if it is.)